src/less/section-title/base/section-title.less

```diff
@@ -1,6 +1,7 @@
 @import "../../mixins/utility/utility-mixins.less";
 
 .section-title {
+    align-items: center;
     display: flex;
     margin: 30px 0 10px 0;
```

Comment:
Maybe it's because of the new align-items, but the top margin looks too large and looks off in the examples. .
I looked in figma and it doesn't seem like it's there.

Reply:
This has been there for a while. For some of the modules we put in a top margin as a "sensible default" for some breathing room from whatever is above it.